### ISSUE TITLE
HTTP403: FORBIDDEN when loading Bulma via CDN in Edge 

### ISSUE THREAD

**Issue by LaundroMat** [jgthms/bulma#1345]
This is about **Bulma**.

### Overview of the problem

I'm using Bulma **version** [0.6.0]
My **browser** is: Edge (latest version, can't find version information ಠ_ಠ)

### Description

When loading Bulma via cdnjs , a 403 is returned:

    HTTP403: FORBIDDEN - The server understood the request, but is refusing to fulfil it.
    (XHR)GET - https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.0/sass/base/minireset.sass

    Could not locate https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.0/sass/utilities/animations.sass specified in source map https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.0/css/bulma.min.css.map.

### Steps to Reproduce

Load this HTML into Edge and open the dev console:

```
<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title></title>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.0/css/bulma.min.css">
</head>
<body>
</body>
</html>
```

**Comment by jgthms** [comment]
Why is it trying to load the Sass file?

**Comment by ctolkien** [comment]
Are there sourcemaps referenced in that file? It's probably looking for them when you open dev tools.

**Comment by ctolkien** [comment]
@LaundroMat  - winkey + r -> 'winver', and report the version number please.

**Comment by jgthms** [comment]
Indeed. Must be the source map.

**Comment by LaundroMat** [comment]
I've updated it with the version number.

**Comment by ctolkien** [comment]
Yep, so if you load the source map file, it references the raw sass files, if you try and request those, the CDN returns a HTTP403.

So this is probably working as expected?

FWIW @LaundroMat  - You're on Edge15 - Edge version 16 came out last week with 1709.

**Comment by Serverfrog** [comment]
I can even get an 403 if I go to the Documentation site. Even with an Proxy between me and the Site.
Real URL [https://bulma.io/documentation/columns/basics/](url)
Proxy [https://ixquick-proxy.com/do/spg/proxy?ep=466b305946434e734d783841544574554d58644350553144464377774930634358566f524b47784a6446465651577069466c394866683046487a4e55624568454843592b636b677a584664554c447845626c46455269592b4f563948666c74594e53524d626c594141416b7a4e676b635730735361786348636b6b5966414166456c776456456b4b4d535656634659664857416d49413162533078574b79565259554a414843772b4f6c5552567864455044425459303044587a73684a787358534155616148644d6446565653797779616b7054565578484b53464750525544587a73684a786b555441564150444d486256465651696b394e6830715756734b6158644d644656565869342f4d4138555831304b5054525564465a47576d6b3849776f46573156545a43465462305a415154774f4a423855536c7466667a785663465641584367344f5238575630315a4c5777514a6b6852516a38684f306363585235614c534652626b3859416d6b6b616849425445684566474a674a52646a463330584e51385a56566b5a4d4434454d6d4e425853776b4f68386254466c444d4435504a52646a555341394968636253783046487a4e41633078475157706a4556774158466c444f4777554e555954555834794e6b7354446c774159576b584f4230574369356d5942394257776b43594759564d514e5851694169616b74545445734b4641563064327068565830634578304e64586b53616855454d324544515439734e45684d58466b434f6d63544f5541534333316759786848446c6c55615449514e684951413330795a553554586b70485a47414863465a574433343d&ek=6479556p4q6o3952563370314s44673357564568&ekdata=0dd217ed3e8e449a31a1f3a87a671fca](url)

**Comment by LaundroMat** [comment]
@ctolkien : even if it's expected behaviour, isn't it a bit confusing for users? After all, it never feels right being confronted with an error message.

**Comment by ctolkien** [comment]
>even if it's expected behaviour, isn't it a bit confusing for users

Agreed. So are the options:

* Remove sourcemap references from the version of Bulma shipped via the CDN's
or
* Make them actually work.

I've no idea on the CDN infrastructure to know if the later is possible.

**Comment by hortonelectric** [comment]
WTF is this.... Bulma does not even load in MS Edge at all for me either. What is the workaround for this "browser"?

**Comment by hortonelectric** [comment]
Am I missing something? Why doesn't Edge load Bulma, but it loads everything else? This is breaking my website in Edge, so far have not figured out why. Maybe just a message blocking Edge saying to download a web browser?

**Comment by ctolkien** [comment]
>Why doesn't Edge load Bulma

Edge does load Bulma, the specific issue we're discussing here is the compiled assets on the CDN still reference sourcemap files which are _not_ present on the CDN. If you open Edge dev tools, it will try and request them - but they don't exist.

This should not impact Bulma from working in any way.
